app/jobs/regular/notify_mailing_list_subscribers.rb

```diff
@@ -5,8 +5,21 @@ module Jobs
   class NotifyMailingListSubscribers < Jobs::Base
     include Skippable
 
+    RETRY_TIMES = [5.minute, 15.minute, 30.minute, 45.minute, 90.minute, 180.minute, 300.minute]
```

Comment:
I think we should use the same pattern used [here](https://github.com/discourse/discourse/blob/master/app/jobs/regular/user_email.rb#L166).

Reply:
The pattern there was simple. I do no think it would be the same in this case.

I could use something like `(count + 1) * 15.minutes`? It would have a different meaning, but the first few values match pretty good.

Reply:
Hmm after reading the topic, why do we need to retry 7 times at those particular intervals? If the job raised an exception, [Sidekiq will add it to the fail queue and retry](https://github.com/mperham/sidekiq/wiki/Error-Handling).

Reply:
Since it's a temporary issue, we're just bypassing the "raise an exception" part ;) No need to fill the logs if we can just retry in a bit.